src/System.Drawing.Primitives/tests/SizeFTests.netcoreapp.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using Xunit;
+
+namespace System.Drawing.PrimitivesTest
+{
+    public partial class SizeFTests
+    {
+        #region SizeF * float tests
+
+        private void MultSizeFFloatTester(float width, float height, float multiplier)
+        {
+            SizeF sz = new SizeF(width, height);
+            SizeF mulExpected;
+
+            mulExpected = new SizeF(width * multiplier, height * multiplier);
+
+            Assert.Equal(mulExpected, sz * multiplier);
+            Assert.Equal(mulExpected, multiplier * sz);
+        }
+
+        [Theory]
+        [InlineData(1000.234f, 2400.933f)]
+        [InlineData(float.MaxValue, 0.0f)]
+        [InlineData(float.MaxValue, 1.0f)]
+        [InlineData(float.MaxValue, -1.0f)]
+        [InlineData(float.MaxValue, float.MaxValue)]
+        [InlineData(float.MinValue, 0.0f)]
+        [InlineData(float.MinValue, 1.0f)]
+        [InlineData(float.MinValue, -1.0f)]
+        [InlineData(float.MinValue, float.MinValue)]
+        [InlineData(float.MaxValue, float.MinValue)]
+        [InlineData(float.MinValue, float.MaxValue)]
+        public void MultiplicationTest(float value1, float value2)
+        {
+            MultSizeFFloatTester(value1, 1.0f, value2);
```

Comment:
Sounds like a solution... and I wouldn't agree more about being concerned as a maintainer of the code.

> MultiplicationTest has two vaguely-named ......... will allow clearer parameter names to be used
I was trying to avoid bloating the main function with various combinations. But please read next point...

>Again, there are no test .......... which passes different values for width and height.
This sounds like a solution. I was more concerned about the said typo that I almost did.

So I will revert back to the original test cases and add special test for different parameters, sort of (3.0f, 4.0f, 5.0f).

Between, I tried resolving a merge conflict and it seems it has messed something up. I am in process of fixing it up. Thanks!